docker-compose.yml

```
@@ -53,13 +63,19 @@ services:
     command: gunicorn backend.wsgi:application
     depends_on:
       - database
+    healthcheck:
+      test: ["CMD-SHELL", "curl --silent --fail http://localhost:8000/api/v2/healthcheck/?format=json"]
+      interval: 10s
+      timeout: 50s
+      retries: 5
```

This is inefficient, as we spawn an extra shell process. Use this:

```suggestion
    healthcheck:
      test: ["CMD", "curl", "--silent", "--fail", "http://localhost:8000/api/v2/healthcheck/?format=json"]
      interval: 10s
      timeout: 50s
      retries: 5
```